bpfman/src/bin/cli/image.rs

```diff
@@ -107,13 +107,22 @@ pub(crate) async fn execute_build(args: &BuildBytecodeArgs) -> anyhow::Result<()
         .unwrap()
         .split('=')
         .collect();
+
     let bc_file = PathBuf::from(first_arg[1]);
-    let (prog_labels, map_labels) = if first_arg[0].contains("EL") {
-        build_bpf_info_image_labels(&bc_file, Some(Endianness::Little))?
+
+    // Make sure bytecode matches host endian if we're building a host endian image
+    // otherwise determine correct endianness based on build argument naming scheme
+    let expected_endianess = if build_context.platforms.is_none() {
+        Endianness::default()
+    } else if first_arg[0].contains("EL") {
```

Comment:
Does this need to be the following (based on the other change you made to allow --bytecode to work for me)?

```suggestion
    } else if first_arg[0].contains("EL") || first_arg[0].contains("BYTECODE_FILE") {
```

Reply:
Good catch, let me reanalyze this logic really quick

Reply:
@Billy99, what was the use case that you think won't work?  The following seems to work for me without the change:
```
bpfman image build --bytecode ./go-kprobe-counter/bpf_x86_bpfel.o -f /home/afredette/go/src/github.com/bpfman/bpfman/Containerfile.bytecode --tag quay.io/afredette/go-kprobe-counter
```

Reply:
While reviewing #1141, a similar command as you entered was failing for me (using --bytecode). Andrew fixed in #1141 with:
https://github.com/bpfman/bpfman/pull/1141/files#diff-2229a2342526fa26a90e753aff04cbe5e9b50fe2ca0c302d208447c9f5148ab7R296
So I was asking if a similar fix was needed here.

Reply:
I'm going to leave this as it is.  Everything seems to work, but we can look at it later.